src/components/PageBlocks/Wrap/Card.vue

```diff
@@ -15,12 +15,26 @@
         <div
           v-if="!headerSet"
         >
-          <h5
+          <div
```

Comment:
This should be the h5 div, if you compare with recordListbase and previous implementation of this file.

Reply:
I don't understand what needs to be done here.